src/devices/bus/coco/coco_t4426.cpp

```diff
@@ -39,11 +39,15 @@
  *
  ***************************************************************************/
 
+#include "emu.h"
+#include "coco_t4426.h"
+#include "includes/coco.h"
+
 #define VERBOSE 0
 
-#define LOGPRINT(x) do { if (VERBOSE) logerror x; } while (0)
-#define LOG(x)      {} LOGPRINT(x)
-#define LOGSETUP(x) {}
+#define LOGPRINT(...) do { if (VERBOSE) logerror(__VA_ARGS__); } while (0)
+#define LOG(...)      {} LOGPRINT(__VA_ARGS__)
```

Comment:
intuition is subjective I guess :) I'll go over it and see if can get improve the intuitiveness here, like remove the brackets and such